compute/admin_guide/vulnerability_management/detect_vulns_unpackaged_software.adoc

```diff
@@ -1,26 +1,40 @@
 == Detect vulnerabilities in unpackaged software
 
-Typically, software is added to container images and hosts with a package manager, such as apt, yum, npm.
-Prisma Cloud has a diverse set of upstream vulnerability data sources covering many different package managers across operating systems, including coverage for Go, Java, Node.js, Python, and Ruby components.
-Prisma Cloud typically uses the package manager’s metadata to discover installed components and versions, comparing this data to the data in the Intelligence Stream's realtime CVE feed.
+Typically, the software is added to container images and hosts with a package manager, such as `apt`, `yum`, and `npm`.
+Prisma Cloud has a diverse set of upstream vulnerability data sources covering many different package managers across operating systems, including coverage for `Go`, `Java`, `Node.js`, `Python`, and `Ruby` components.
+Prisma Cloud typically uses the package manager's metadata to discover installed components and versions, comparing this data to the data in the Intelligence Stream's real-time CVE feed.
+The host defender only scans the running applications that are installed with a package manager on the host.
 
 Sometimes, you might install software without a package manager.
-For example, software might be built from source and then added to an image with the Dockerfile `ADD` instruction or your developers might unzip software from a tar ball to a location on a host, and utilize the application.
+For example, the software might be built from source and then added to an image with the Dockerfile `ADD` instruction, or your developers might unzip software from a tarball to a location on a host, and utilize the application.
 In these cases, there is no package manager data associated with the application.
 
-Prisma Cloud uses a variety of analysis techniques to detect metadata about software not installed by package managers. These are purpose built differently for images and hosts. 
+Prisma Cloud uses a variety of analysis techniques to detect metadata about software not installed by package managers. These are purpose-built differently for images and hosts.
+We scan the entire root volume In Agentless, and can detect third-party packages in addition to OS packages.
```

Comment:
this sentence  separates the flow with the next sentence, and does not have context. 
At a minimum can we move it to be the last sentence?
Can we revise it to say " If you have configured agentless scanning(add link), the entire root volume on the host is scanned, and vulnerabilities on third-party packages and OS packages are detected.


@Aqsaxvv092 , is this correct?